index.rst

```diff
@@ -6,24 +6,219 @@
 
 .. TODO: Delete the note below before merging new content to the main branch.
 
-.. note::
-
-   **This technote is a work-in-progress.**
-
 Abstract
 ========
 
-This is a technote for the description of results from Individual Hardpoint Breakaway Test. 
+This technote describes the analysis and results derived from individual hard point breakaway tests conducted with the M1M3 surrogate.
+The notebook is located within `notebooks_vandv <https://github.com/lsst-sitcom/notebooks_vandv/blob/tickets/SITCOM-838/notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-838_Anaysis.ipynb>`_ GitHub repository.
+With all the steps guarded with reasonable timeouts, so problems are detected if hard point cannot travel to reach low or high limit switches, etc.
+**If this test shows that the hard points do not work properly at the limits, this could be one of the blockers for the installation of M1M3 until it is solved.** 
+
+
+Hardpoint Breakaway Test
+========================
+
+The active support system of the M1M3 includes six axial hard point actutators in the hexapod configuration. :cite:`2018SPIE10700E..3GD`
+These hard point actuators should minimize forces during slews at any TMA position and be kept under the breakaway limit. 
```

Comment:
I would confirm this sentence with Petr. From what I understand, the 156 Force Actuators are responsible for keeping the forces in the Hard Points small. The HPs are only responsible for guiding the rigid body motion of the mirror, and the forces applied to them should be within the range you describe later. It might be my misunderstanding, but this is a good opportunity to review and clarify it.